packages/contracts/src/utils/order_factory.ts

```diff
@@ -1,36 +1,35 @@
-import { Order, SignedOrder, ZeroEx } from '0x.js';
+import { ZeroEx } from '0x.js';
 import { BigNumber } from '@0xproject/utils';
-import { Web3Wrapper } from '@0xproject/web3-wrapper';
 import * as _ from 'lodash';
 
-import { DefaultOrderParams } from './types';
+import { orderUtils } from './order_utils';
+import { signingUtils } from './signing_utils';
+import { DefaultOrderParams, SignatureType, SignedOrder, UnsignedOrder } from './types';
 
 export class OrderFactory {
-    private _defaultOrderParams: Partial<Order>;
-    private _zeroEx: ZeroEx;
-    constructor(zeroEx: ZeroEx, defaultOrderParams: Partial<Order>) {
+    private _defaultOrderParams: Partial<UnsignedOrder>;
+    private _secretKey: Buffer;
+    constructor(secretKey: Buffer, defaultOrderParams: Partial<UnsignedOrder>) {
         this._defaultOrderParams = defaultOrderParams;
-        this._zeroEx = zeroEx;
+        this._secretKey = secretKey;
```

Comment:
If we just use regular TestRPC, we can just store the private key here, and use a HookedWalletSubprovider to sign things.